extensions/amp-ima-video/0.1/validator-amp-ima-video.protoascii

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-ima-video
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-ima-video extension .js script"
+  requires: "amp-ima-video"
+  extension_spec: {
+    name: "amp-ima-video"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-ima-video>
+  html_format: AMP
+  tag_name: "AMP-IMA-VIDEO"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  satisfies: "amp-ima-video"
+  requires: "amp-ima-video extension .js script"
+  # data-* is generally allowed, but it's not generally mandatory.
+  attrs: {
+    name: "data-src"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "https"
+      allow_relative: true  # Will be set to false at a future date.
+    }
+    blacklisted_value_regex: "__amp_source_origin"
+  }
+  attrs: {
+    name: "data-tag"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "https"
+      allow_relative: true  # Will be set to false at a future date.
+    }
+  }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/media/amp-ima-video"
```

Comment:
One last little thing, the URL structure changed again for ampproject :) It's no longer needed to list the sub-component, please remove `media/`:
`spec_url: "https://www.ampproject.org/docs/reference/components/amp-ima-video"`